Makefile

```diff
@@ -107,7 +102,7 @@ build.examples.emscripten: $(emscripten_sources)
 %/cat.wasm : cargo_target := wasm32-wasi
 
 .PHONY: build.examples.rust
-build.examples.rust: examples/allocation/rust/testdata/greet.wasm examples/wasi/testdata/cargo-wasi/cat.wasm
+build.examples.rust: examples/allocation/rust/testdata/greet.wasm imports/wasi_snapshot_preview1/example/testdata/cargo-wasi/cat.wasm
 
 # Builds rust using cargo normally, or cargo-wasi.
 %.wasm: %.rs
```

Comment:
not relevant to this PR, but I realized that `%.wasm` defined twice :D

Reply:
yeah good point